src/Workspaces/Core/MSBuild/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj

```diff
@@ -28,6 +28,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildVersion)" ExcludeAssets="Runtime" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="[$(MicrosoftBuildVersion),)" ExcludeAssets="Runtime" />
```

Comment:
@jmarolf how is this different than just writing `Version="$(MicrosoftBuildVersion)"`?

Reply:
just tested this. You are right, they are equivalent